hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenGrantKycToAccountHandlerTest.java

```diff
@@ -162,12 +165,29 @@ class HandleTests {
         @Mock
         private WritableTokenRelationStore tokenRelStore;
 
+        @Mock
+        private ReadableTokenStore readableTokenStore;
+
+        private static final AccountID TREASURY_ACCOUNT_9876 = BaseCryptoHandler.asAccount(9876);
+        private static final TokenID TOKEN_531 = BaseTokenHandler.asToken(531);
+
+        private Token newToken531() {
```

Comment:
Nit: Token is immutable. It would be ok to use a constant here.

Reply:
Fixed.